tui/bundle.go

```diff
@@ -44,41 +45,40 @@ func (bp *BundlePage) GetConfiguredValue() string {
 // Activate marks the checkbox selections based on the data model
 func (bp *BundlePage) Activate() {
 	bp.GetWindow().SetVisible(true)
-	model := bp.getModel()
-
-	// Network connection is required to add additional bundles
-	if !controller.NetworkPassing {
-		if dialog, err := CreateNetworkTestDialogBox(bp.tui.model); err == nil {
-			if dialog.RunNetworkTest() {
-				// Automatically close if it worked
-				clui.RefreshScreen()
-				time.Sleep(time.Second)
-				dialog.Close()
-			}
-		}
-	}
-
-	for _, curr := range bundles {
-		state := 0
-
-		if model.ContainsUserBundle(curr.bundle.Name) {
-			state = 1
-		}
-
-		curr.check.SetEnabled(controller.NetworkPassing)
-		curr.check.SetState(state)
-	}
 
 	if controller.NetworkPassing {
 		bp.offlineLabel.SetTitle("")
-	} else {
-		bp.offlineLabel.SetTitle("Network check failed.")
 	}
 
 	bp.confirmBtn.SetEnabled(controller.NetworkPassing)
 	bp.offlineLabel.SetEnabled(!controller.NetworkPassing)
 }
 
+func bundleCheck(bp *BundlePage) {
+	if !controller.NetworkPassing {
+		text := utils.Locale.Get("This requires a working network connection. Proceed with a network test?")
```

Comment:
The Text-based UI we have not been supporting other languages; mainly due to highly limited screen space and layout issues.
Drop the translation here, and then added a newline between the sentence and the question please so that the question isn't wrapped in the box.

Reply:
Done